src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -184,8 +184,8 @@ <h5><span jhiTranslate="artemisApp.studentExamDetail.exercises">Exercises</span>
                                     (exercise?.studentParticipations)![0].results?.length > 0 &&
                                     (exercise?.studentParticipations)![0].results[0].hasComplaint
                                 "
-                                >Yes</span
-                            >
+                                jhiTranslate="artemisApp.studentExamDetail.hasComplaintYes"
```

Comment:
We should probably use `global.generic.yes` here instead.

Reply:
done!